src/common/utils/getUrlParams.js

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
https://github.com/geops/mobility-toolbox-js/blob/master/package.json#L17

Please use `query-string`, which is already a dependency.